index.js

```diff
@@ -32,28 +32,19 @@ module.exports = function (dir, opts) {
         pump(socket, client.createRpcStream({ref: socket}), socket, function () {
             if (!client.isOpen()) return;
 
-            var db = level(dir, opts);
+            var db = level(dir, opts, onopen);
```

Comment:
This (passing a callback) is needed because there's a bug in `levelup` ever since we introduced promise support (https://github.com/Level/levelup/issues/656).

Reply:
Isn't it fixed now?

Reply:
Yes, you're right. I now like the callback though, it's cleaner anyway (one function instead of two, no need to remove listeners, no need to check the type of the error).